ddtrace/tracer/tracer.go

```diff
@@ -143,24 +170,50 @@ func newTracer(opts ...StartOption) *tracer {
 		prioritySampling: newPrioritySampler(),
 		pid:              strconv.Itoa(os.Getpid()),
 	}
+	t.config.statsd.Incr("datadog.tracer.started", nil, 1)
 	if c.runtimeMetrics {
-		statsd, err := statsd.NewBuffered(t.config.dogstatsdAddr, 40)
-		if err != nil {
-			log.Warn("Runtime metrics disabled: %v", err)
-		} else {
-			log.Debug("Runtime metrics enabled.")
-			go t.reportMetrics(statsd, defaultMetricsReportInterval)
-		}
+		log.Debug("Runtime metrics enabled.")
+		t.wg.Add(1)
+		go t.reportMetrics(defaultMetricsReportInterval)
 	}
-
+	t.wg.Add(2)
 	go t.worker()
-
+	go t.statsSender()
 	return t
 }
 
+func (t *tracer) sendSpanStats() {
```

Comment:
If we rename `statsSender` to `reportHealthMetrics` maybe we can make this `sendHealthMetrics`? Lastly, can all of this be moved into `metrics.go`. I think it makes sense for them to be there.